### DISCUSSION TITLE
be注册不上

### DISCUSSION THREAD

**Discussion by whwususu** [apache/doris#34770] [A - General / Q&A] [answered]
我用docker的方式部署集群，FE三个已经正常启动，BE也是docker启动，
![image](https://github.com/apache/doris/assets/25104771/ccfeadbc-b74c-4ea9-9926-04bc41ffc036)
![image](https://github.com/apache/doris/assets/25104771/112bc176-3309-4313-8f26-dab4c47420be)
BE连初始化参数都不行，
![image](https://github.com/apache/doris/assets/25104771/037a7ee5-8a96-4d4b-9c33-303488471cff)
![image](https://github.com/apache/doris/assets/25104771/518b643e-366a-43d7-bd5f-6cc605dcc262)
FE正常，谢谢！

**Comment by yagagagaga** [comment, marked as answer]
Maybe you can try apache/doris:2.0.3-fe-x86_64 and apache/doris:2.0.3-be-x86_64

**Reply by whwususu** [reply]
apache/doris:2.1.2-fe-x86_64 and apache/doris:2.1.2-be-x86_64  依旧不行

**Comment by whwususu** [comment]
![image](https://github.com/apache/doris/assets/25104771/f9c38fcf-3ed0-468b-bc5f-7a33540bf7dc)
容器启动这个命令
![image](https://github.com/apache/doris/assets/25104771/d9e206d2-cb51-449f-bd5b-0e88fba5bbff)
